PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -1295,11 +1295,14 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return MAKELONG(auxVer, mainVer);
 		}
 
+		case NPPM_GETCURRENTMACROSTATUS:
 		case WM_GETCURRENTMACROSTATUS:

```

Comment:
If `NPPM_GETCURRENTMACROSTATUS` is used, then `WM_GETCURRENTMACROSTATUS` can be removed.

Reply:
Well, this is an internal code I wasn't sure if I could tamper with. So I decided to left it untouched for now (I was afraid to break something so I never deleted anything, just added new funcionality).

Reply:
Done that.